src/main.c

```diff
@@ -278,7 +279,7 @@ int main(int argc, char **argv)
 		return EXIT_FAILURE;
 	}
 
-	upnp_renderer_set_mime_filter(mime_filter);
+	connmgr_set_mime_filter(mime_filter);
```

Comment:
^^ can you add more details to the mime_filter flag description ? It will help discoverability of the `+` and `-` feature.